assets/src/edit-story/components/library/libraryPanes.js

```diff
@@ -35,15 +33,12 @@ import { TextPane } from './panes/text';
 import { ElementsPane } from './panes/elements';
 import { PageTemplatesPane } from './panes/pageTemplates';
 
-function LibraryPanes() {
+function LibraryPanes({ mediaPaneStyles, textPaneStyles }) {
```

Comment:
I don't love this api. Maybe we should pass these in as one prop? Or something?

Reply:
Kinda feels like 6  or half a dozen to me. Right now, this reads really easily - it's very clear what's getting passed in and what it ties to.